config/migrations/dev/2023/20230119110751-add-read-only-module-flag-to-bestuurseenheid.sparql

```diff
@@ -0,0 +1,19 @@
+# Adding viewOnly flag to the bestuurseenheid who has access to the eredienst modules
```

Comment:
don't commit this file, this will run in production. `mkdir config/dev-migrations/2023/` and add a README.md there that this won't run by default, and if you need these that you'll have to run them manually.